### DISCUSSION TITLE
Is there a way to delete a channel created for realtime subscription?

### DISCUSSION THREAD

**Discussion by shankar-narayanan-d** [supabase/supabase#12282] [Questions] [answered]
The function `supabase.removeAllChannels()` mentioned in the docs says, unused channels may cause degradation as more clients are simultaneously subscribed.

So is it possible to delete a particular channel.

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/reference/javascript/removechannel

I’m not sure if channel name is the assigned variable or the name in the channel….
const channel = supabase
  .channel('db-changes')
